test/test_autograd.py

```diff
@@ -1502,6 +1502,66 @@ def func(x):
         gradcheck(func, [x])
         gradgradcheck(func, [x])
 
+    def test_complex_views(self):
```

Comment:
Can you add a couple comments in this function to explain what you're testing and why these are special cased here.